<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background-color:rgb(0, 158, 188,220);
color:black;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>230</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color:rgb(183, 153, 20,150);
color:black;
}
QLabel{
background-color:rgba(255, 255, 255, 0);
color:black;
font: 13pt &quot;等线&quot;;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 50pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Chat</string>
     </property>
    </widget>
    <widget class="QListWidget" name="ID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>211</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
font: 18pt &quot;MV Boli&quot;;
background-color:rgb(199, 199, 199);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>联系人：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QLabel" name="SendName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="SendMessage">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>470</y>
      <width>611</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-radius:30px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#ced1d8;
border-style:outset;
border-radius:10px;
font: 14pt &quot;MV Boli&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Send">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>620</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#ced1d8;
border-style:outset;
border-radius:10px;
font: 14pt &quot;MV Boli&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>771</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 35pt &quot;MV Boli&quot;;</string>
    </property>
    <property name="text">
     <string>      Start to chat!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ReceiveMessage">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>751</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-radius:30px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
